tests/test_pipenv.py

```diff
@@ -1120,7 +1112,6 @@ def test_clean_on_empty_venv(self, pypi):
             c = p.pipenv('clean')
             assert c.return_code == 0
 
-
     @pytest.mark.install
     def test_environment_variable_value_does_not_change_hash(self, pypi, monkeypatch):
```

Comment:
is there some reason this test failure didn't bubble up from CI?

Reply:
I'm kind of confused about why there is a need to use monkeypatch here.   Is that actually preferable over using the `temp_environ` context manager? If so, why?

Reply:
It doesn't have a mark that is included in TEST_SUITE env
You may need to add `@pytest.mark.run` if it is proper for your case